static_frame/core/quilt.py

```diff
@@ -466,29 +476,67 @@ def rename(self, name: NameType) -> 'Quilt':
 
     #---------------------------------------------------------------------------
 
-    def __repr__(self) -> str:
-        '''Provide a display of the :obj:`Quilt` that does not realize the entire :obj:`Frame`.
-        '''
-        if self.name:
-            header = f'{self.__class__.__name__}: {self.name}'
-        else:
-            header = self.__class__.__name__
-        return f'<{header} at {hex(id(self))}>'
-
+    @doc_inject()
     def display(self,
             config: tp.Optional[DisplayConfig] = None,
             *,
             style_config: tp.Optional[StyleConfig] = None,
             ) -> Display:
-        '''Provide a :obj:`Frame`-style display of the :obj:`Quilt`.
+        '''{doc}
+
+        Args:
+            {config}
         '''
         if self._assign_axis:
             self._update_axis_labels()
-        return self.to_frame().display( #type: ignore
-                config,
+
+        bus_columns = False # Flag to strip off column dtype info
```

Comment:
Maybe this can have a more explicit name, such as `drop_first_row_last_item` or similar.